docs/modules/data-connections/pages/data-connections-configuration.adoc

```diff
@@ -13,6 +13,10 @@ To configure a data connection to an external system, you must do the following:
 * Provide a unique identifier (`name`) for the data connection.
 * Choose the correct type of data connection for your external system. 
 
+See examples for other <<configuration-options, connection properties>> for each data connection type.
+
+=== Example JDBC Data Connection
+
 This example configuration shows data connections to both a MySQL database and a H2 data store. Both data connections use a JDBC connection.
 
 [tabs]
```

Comment:
Should we also add SQL tab? we can create the data connection also using SQL.

One caveat though is that those behave slightly differently:
- then can be replaced or dropped
- they don't survive the cluster restart

Reply:
I've added a SQL tab for each example along with some intro text to cover your comments.